spec

```diff
@@ -405,7 +405,7 @@ beginseg
     include "build/src/code/fault.o"
     include "build/data/fault.bss.o"
     include "build/src/code/fault_drawer.o"
-    include "build/asm/code_800D71F0.o"
+    include "build/asm/sjis.o"
```

Comment:
I'm not sure about the filename `sjis`, it seems to abbreviated. That's suppose to stand for `Shift-JIS` right? Perhaps `shift_jis.s`. Alternatively, you use the `Kanji_` prefix in the function name, so what about `kanji.s`?

Reply:
Personally I dont like just `kanji.s` because this function is  specific for the Shift-JIS encoding.

Kanji symbols (glyphs? characters?) are be encoded  differently in different encodings. 
For example, the original debug strings in the repo are encoded in EUC-JP (but we save the C files in UTF-8 and re-encode them during build), but the in-game japanese text is encoded in a modified Shift-JIS encoding (i know they added hylian symbols, replacing something else among other changes)

In summary (i didn't expect to write so much...), I think this file should somehow explicitly mention the Shift-JIS encoding.

Reply:
Yeah, this file is a sort of Shift-JIS++ text -> kanji offsets converter, and can only be used like that (if you want to convert EUC-JP, you'd have to convert it to JIS X 0208 and then Shift-JIS first, for example.

`kanji` is not precisely Shift-JIS, it has some symbols missing and, as @AngheloAlf says, some extra stuff like the Hylian (which I assume is not used at all, since `kanji` is not present on PAL), but it's it's probably about 90% the same as the particular section of Shift-JIS that it covers.

Reply:
Renamed it based on the original name of the n64dd library file that contains this function.